sdk/Sdk.Generators/Extensions/StringExtensions.cs

```diff
@@ -29,5 +30,20 @@ public static string TrimStringsFromEnd(this string str, IReadOnlyList<string> s
 
             return result;
         }
+
+        /// <summary>
+        /// Returns a copy of the string where the first character is in lower case.
+        /// </summary>
+        public static string ToLowerFirstCharacter(this string str)
```

Comment:
nit:
```suggestion
        public static string ToCamelCase(this string str)
```

Reply:
I am not sure "ToCameCase" is appropriate name as we are not doing that. for example : for input "HTTPTRIGGER" , the ideal camelCase result should be "httpTrigger", but we would return "hTTPTRIGGER"

Reply:
^ This was also my reasoning.

Reply:
Good points